pkg/sdk/dynamic_table.go

```diff
@@ -22,6 +22,8 @@ type createDynamicTableOptions struct {
 	dynamicTable bool                    `ddl:"static" sql:"DYNAMIC TABLE"`
 	name         SchemaObjectIdentifier  `ddl:"identifier"`
 	targetLag    TargetLag               `ddl:"parameter,no_quotes" sql:"TARGET_LAG"`
+	Initialize   *string                 `ddl:"parameter,no_quotes" sql:"INITIALIZE"`
```

Comment:
Please introduce types for RefreshMode and Initialize (e.g. like `ColumnConstraintType`).

Then you can have the AllRefreshModes list (like `AllColumnConstraintTypes`) which can be used for validation (`ValidateFunc`) on the resource parameter (like in `table_constraint.go`)

Reply:
Will do - I actually created these types in my first iteration of working on this but removed them after I felt like they didn't actually add value or readability to the code. But I also don't know yall's conventions and figured you'd let me know in the PR review.

Reply:
Thanks for adding them. Please use them in multiple places:
- in `createDynamicTableOptions` (instead of `*string` for `Initialize` and `RefreshMode`)
- in acceptance tests (instead of hardcoded string values)
- in resource implementation (instead of hardcoded string values)